kotlin-jvm/build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
     kotlin("kapt")
     kotlin("plugin.serialization")
     id("com.squareup.sqldelight")
+    id("com.apollographql.apollo").version("2.4.1")
```

Comment:
You can remove the version here, now that you have put ` plugin.com.apollographql.apollo=2.4.1 ` in `versions.properties`

Reply:
Ah right, Thank 👍